internal/check/valid_owner.go

```diff
@@ -314,15 +321,18 @@ func (v *ValidOwner) validateGitHubUser(ctx context.Context, name string) *valid
 	}
 
 	_, isMember := (*v.orgMembers)[userName]
-	if !isMember {
-		return newValidateError("User %q is not a member of the organization", name)
+	_, isOutsideCollaborator := (*v.outsideCollaborators)[userName]
+	if !(isMember || isOutsideCollaborator) {
+		return newValidateError("User %q is not an owner of the repository", name)
```

Comment:
owner in the context of the repository means a bit more 🙂 
```suggestion
		return newValidateError("The user %q is neither a collaborator nor a member of this repository.", name)
```